### DISCUSSION TITLE
Pop-up placed to the right of the event.

### DISCUSSION THREAD

**Discussion by HHXXXH** [vkurko/calendar#152] [Q&A] [answered]
I want to create a popup with the event details on the right-hand side when I click on an event.I have been able to create a table that appears when I click on an event using `eventClick()`.However, I cannot move the table to the right side of the clicked event.

Is there a better way to do this?I am a beginner and would appreciate a demonstration.

**Comment by vkurko** [comment]
Hi, maybe this old [comment](https://github.com/vkurko/calendar/issues/17#issuecomment-902483944) will help you with the issue.

**Comment by HHXXXH** [comment]
```
        eventContent: function(info) {
          return info.event.display === 'auto'
            ? '<div class="ec-event-time">' + info.timeText + '</div>' +
              '<div class="ec-event-title">' + info.event.title + '</div>' +
              '<div hidden class="popover-content">Popover content: ' + info.timeText + '</div>'
            : '';
        },
```
Is this part of the code correct?
In my environment(svelte), the HTML text is displayed as it is on the event.

**Reply by HHXXXH** [reply]
This part of the problem has been resolved.
```
        eventContent: function(info) {
          return info.event.display === 'auto'
            ? {html: '<div class="ec-event-time">' + info.timeText + '</div>' +
              '<div class="ec-event-title">' + info.event.title + '</div>' +
              '<div hidden class="popover-content">Popover content: ' + info.timeText + '</div>' }
            : '';
        },
```

**Comment by HHXXXH** [comment]
I am struggling with the point of displaying the popup on the right side, I changed the processing in  `eventDidMount` function as follows but it did not work. (This code shows the event in the 0:00 to 1:00 block of the day on which it is clicked.) This question may be an unrelated issue to this repository, but it would be great to get an answer.

```
      eventDidMount: function (info) {
        if (info.event.display === 'auto') {
          jQuery(info.el).on('click', function () {
            const popupContent = jQuery(this).find('.popup-content').html();
            const popup = jQuery('<div class="popup"></div>');
            popup.html(popupContent);
            popup.insertAfter(this);
          });
        }
      }
```

**Comment by vkurko** [comment]
If you are going to continue with bootstrap 4.6, then you need to look at the settings for [popover](https://getbootstrap.com/docs/4.6/components/popovers/). For example, there is a `trigger` option, which can be set to `click`, and then the pop-up window will appear on click. Apparently, if you want the window to close on click outside of the event, then you must implement it yourself.

**Comment by HHXXXH** [comment, marked as answer]
We decided to populate the event in a different way. Thank you for your response. If you feel this question is less relevant to this repository, please delete it.
